cfme/containers/image_registry.py

```diff
@@ -26,6 +28,10 @@ def __init__(self, host, provider):
         self.host = host
         self.provider = provider
 
+    def _on_detail_page(self):
+        return sel.is_displayed(
+            '//div//h1[contains(., "{} (Summary)")]'.format(self.host))
```

Comment:
please, do not use this idiom anymore. We have `summary_title` and `breadcrumbs` functions in `sel` that do wrap this.